network.go

```diff
@@ -17,6 +17,16 @@ type Network interface {
 	Remove(context.Context) error // removes the network
 }
 
+type DefaultNetwork string
+
+func (n DefaultNetwork) ApplyGenericTo(opts *GenericProviderOptions) {
```

Comment:
Is there any usage of `ApplyGenericTo`? I cannot see it anywhere

Reply:
it's the implementation of the `GenericProviderOption` interface